sftp-server.c

```diff
@@ -807,7 +807,11 @@ process_write(u_int32_t id)
 			error("process_write: seek failed");
 		} else {
 /* XXX ATOMICIO ? */
+#ifdef WINDOWS
```

Comment:
This is a platform level change rather than an application change.

Reply:
I did that initially, but that somehow broke cases that worked before, like when the buffer is the default size. I couldn't trace exactly how that was breaking things, but I believe it was because the write function is called by other parts of the code, and that size limit was there to avoid problems on those other calls.